tests/LocaleTest.php

```diff
@@ -43,13 +44,23 @@ public function testImmutability(): void
         $this->assertNotSame($localeMiddleware->withDefaultLocale('uz'), $localeMiddleware);
         $this->assertNotSame($localeMiddleware->withEnableDetectLocale(true), $localeMiddleware);
         $this->assertNotSame($localeMiddleware->withEnableSaveLocale(false), $localeMiddleware);
-        $this->assertNotSame($localeMiddleware->withLocales(['ru-RU', 'uz-UZ']), $localeMiddleware);
+        $this->assertNotSame($localeMiddleware->withLocales(['ru' => 'ru-RU', 'uz' => 'uz-UZ']), $localeMiddleware);
```

Comment:
This is a bit unusual. There's no phpdoc but it was desinged to accept array of strings originally.

Reply:
phpdoc added

Reply:
constructor only 👀

Reply:
added